docker/utils/socket.py

```diff
@@ -37,8 +37,7 @@ def read(socket, n=4096):
             select.select([socket], [], [])
         else:
             poll = select.poll()
-            poll.register(socket)
-            poll.poll()
```

Comment:
and this line is not needed anymore?

Reply:
Oops, yes, it is. I just deleted it by mistake when I deleted some logs for my testing... I just re-added it now.